test/acceptance/test/utils.go

```diff
@@ -268,6 +277,16 @@ func runCommandAndReturnOutput(commandToRun string) (stdOut string, stdErr strin
 	return string(session.Wait().Out.Contents()), string(session.Wait().Err.Contents())
 }
 
+func generateTestInputs() TestInputs {
```

Comment:
I like this better. One thing to keep in mind is that multiple return values might make things a little more expressive, although 4 return values is probably pushing the limit.

Reply:
I tried that first but then it felt like too many return values and there is good chance that we need more return values in the future. Moreover, it means for every single extra return for new tests, we need to update all tests.